blog/2024-02-25-import_executor.md

```diff
@@ -0,0 +1,18 @@
+---
+author: J. Nick Koston
+authorURL: https://github.com/bdraco
+title: Importing integrations that may block the event loop
+---
+
+Starting from Home Assistant 2024.3, `import_executor` can be specified in the `manifest.json` to facilitate importing an integration in the executor to avoid blocking the event loop. Setting this flag can significantly improve startup stability for integrations that do significant work at import time.
```

Comment:
Is there a reason we should not enable this? If this is new and great, why don't we enable it for all integrations?

Reply:
If it takes 0.00 to import (which will be the case for most small integrations) it ends up creating an executor job that could have been avoided